python/sandbox/das_gate/dasgate.py

```diff
@@ -0,0 +1,94 @@
+from hyperon import *
+from hyperon.ext import register_atoms
+
+import os
+import re
+import json
+
+from hyperon_das import DasAPI
+from hyperon_das.das import QueryOutputFormat as Format
+from hyperon_das.pattern_matcher import (
+    Link,
+    Node,
+    Variable,
+    And,
+    Or,
+    Not,
+    PatternMatchingAnswer,
+)
+
+class DASpace(AbstractSpace):
+
+    def __init__(self, unwrap=True):
+        super().__init__()
```

Comment:
Not technically needed since AbstractSpace is pure-virtual, but also harmless.

Reply:
OK, I see. I'd still keep it here, because if some useful functionality is introduced in the parent class, then it will not be missed in DASpace as well. I'm also ok to remove it if `Abstract` part is a strong hint that no concrete functionality will be ever introduced.